### ISSUE TITLE
Update overlay background color according to Photon

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons#11834]
The overlay background is blue but it should be gray according to the Photon spec: https://design.firefox.com/photon/patterns/overlay.html.

![](https://design.firefox.com/photon/images/overlay/overlay-style.svg)
